imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -12,62 +12,166 @@ import { setDrawerState } from '../general-actions'
 import Units, { collectionName } from '../../api/units'
 import UnitMetaData from '../../api/unit-meta-data'
 import FloatingActionButton from 'material-ui/FloatingActionButton'
+import { Tabs, Tab } from 'material-ui/Tabs'
+import SwipeableViews from 'react-swipeable-views'
+
+class UnitsByRoles extends Component {
+  render () {
+    const { unitsByRoles, showAddBtn, handleUnitClicked, handleAddCaseClicked, administrate } = this.props
+    return (
+      <div>
+        {unitsByRoles.length === 0 ? (undefined) : (
+          <div className='bb b--black-10 bg-very-light-gray f6 fw5 mid-gray pv2 pl2'>
+            { administrate ? ('Units I Administrate') : ('Units Involved In') }
+          </div>
+        )
+        }
+        {
+          unitsByRoles.map(({ id, name, description, metaData }) => (
+            <MenuItem key={id} innerDivStyle={{padding: 0}} onClick={() => handleUnitClicked(id)} >
+              <div className='mt2 ph2 bb b--very-light-gray br1 w-100 flex items-center pa2'>
+                <FontIcon className='material-icons' color='var(--semi-dark-gray)'>home</FontIcon>
+                <div className='ml3 mv1 semi-dark-gray lh-copy flex-grow overflow-hidden'>
+                  <div className='ti1 ellipsis'>{metaData.displayName || name}</div>
+                  <div className='ti1 ellipsis silver'>{ metaData.moreInfo || description}&nbsp;</div>
+                </div>
+                { !showAddBtn && (
```

Comment:
Why is it rendering if `showAddBtn` is false? shouldn't it be the other way around?